### PR TITLE
Fix a mistranslation in set.md.

### PR DESCRIPTION
**Pull request by masaniwasdp**
Change "設定" to "set".

### CODE REVIEW

File: docs/csharp/language-reference/keywords/set.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ---
-title: "設定 (C# リファレンス)"
+title: "set (C# リファレンス)"
```
Comment (mairaw) [inline docs/csharp/language-reference/keywords/set.md]: I imagine you’d have to make the same change on line 21, no?

Comment (masaniwasdp) [reply docs/csharp/language-reference/keywords/set.md]: Yes, I overlooked it.

**Comment by v-radelg** [comment]
Hi @masaniwasdp , we are still in discussing, will update the decision later soon. 
		
Thanks for your support,

Microsoft C&E Localization Team

**Comment by v-radelg** [comment]
@masaniwasdp , Thank you for your feedback and suggestion! 

The suggestion will be merged to the production, we will re-release this articles in the next loc release round.

Microsoft C&E Localization Team

**Pull request merged**